docs/content-query-syntax.md

```diff
@@ -0,0 +1,334 @@
+| **?** &nbsp;&nbsp; single character wildcard search		| **tru?k** &nbsp;&nbsp; returns content containing *truck* or *trunk*			|
+| **&ast;** &nbsp;&nbsp; multiple character wildcard search	| **app*** &nbsp;&nbsp; returns content containing *apple* or *application*		|
+
+## Fuzzy search
+We can search similar words based on Lucene's Fuzzy search. Fuzzy search is a simple term followed by "~" and a small number (0 < n < 1). Fuzzy search uses Levenshtein or Edit Distance algorithm in depth. Example:
+```bash
+Keyword:abbreviate~0.8
+```
+With this query you can find documents that contain a word with one different character from "abbreviate" e.g. abreviate, sbbreviate etc.
+
+## Proximity search
+It is possible to limit the distance of the relevant words in the result documents. The proximity search is two word in quots and followed by "~" and a positive whole number. This number defines the maximal count of words between the given two word. Example:
+```bash
+Description:"Lorem amet"~3
+```
+This query will find the following text: "Lorem ipsum dolor sit amet" because there is 3 words between "Lorem" and "amet".
+
+## Fields
+When performing a search you can specify the field you want to search in. In the previous cases Lucene used the **default field** called *_Text*, which contains all relevant data of a content. If you search for apple, it is the same as *_Text:apple*.
+
+You can name any existing field, the syntax is *FieldName:VALUE*.
+```bash
+DisplayName:about
+```
+There are a few special field types that have their own syntax for defining the query term. For example **DateTime**:
+```bash
+ModificationDate:>'2010-09-01 12:00:00'
+```
+If you want to know more about how fields are indexed, visit the [Field Indexing](https://github.com/SenseNet/sensenet/blob/master/docs/field-indexing.md)  article.
+
+### Common search-related fields
+Sense/Net has lots of built-in fields that you can use to make more precise queries. These fields are related to type or place of the content:
+
+| **Name**		| **Comment**													| **Example**						|
+| -------------	| ------------------------------------------------------------- | ---------------------------------	|
+| **InTree**	| Returns the given folder and all contents under it			| InTree:"/Root/MyContent"			|
+| **InFolder**	| Returns all content in that particular folder					| InFolder:"/Root/MyContent/Media"	|
+| **Type**		| Returns1 all content that is of the given type				| Type:Article						|
+| **TypeIs**	| Returns all content that is of the given type or any subtype	| TypeIs:WebContent					|
+
+> It is usually advised to use quotation marks when querying paths, since paths containing spaces and other special characters like '-' or '_' could cause trouble and "split up" the expression into two separate expressions instead of handling it as one query for path.}}
```

Comment:
```suggestion
> It is usually advised to use quotation marks when querying paths, since paths containing spaces and other special characters like '-' or '\_' could cause trouble and "split up" the expression into two separate expressions instead of handling it as one query for path.}}
```